pages/_app.tsx

```diff
@@ -42,16 +44,19 @@ const DMSApp = ({
   walletToken: string;
 }) => {
   const [loadingUser, setLoadingUser] = useState(true);
-  const [initialUser, setInitialUser] = useState(undefined);
+  const [initialUser, setInitialUser] = useState<WalletUser | undefined>(undefined);
   const { NEXT_PUBLIC_ARRANGER_API_URL } = getConfig();
 
   useEffect(() => {
     if (!initialUser && walletToken) {
-      console.log('Fetching user info');
+      console.info('Fetching user info');
```

Comment:
Let's minimize the amount of console logging noise, unless it serves an important function.